<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSceneController</class>
 <widget class="QWidget" name="QSceneController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>453</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QSceneController</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>208</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_SceneList">
      <property name="text">
       <string>Scene List</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_SceneList"/>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_CreateNewScene">
        <property name="text">
         <string>Create Scene</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_SaveScene">
        <property name="text">
         <string>Save Scene</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_SceneId">
      <property name="text">
       <string>Scene ID</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_SceneID">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_SceneName">
      <property name="text">
       <string>Scene Name</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_SceneName"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_Physics">
      <property name="text">
       <string>Physics</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="checkBox_Physics">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_CameraList">
      <property name="text">
       <string>CameraList</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="comboBox_CameraList"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_CameraSpeed">
      <property name="text">
       <string>Camera Speed</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_CameraSpeed"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
